Makefile

```diff
@@ -19,11 +19,27 @@ tmp/dev_image_id:
 format: prepare
 	${DOCKRUN} bash ./scripts/format.sh
 
-check: prepare
+check: prepare format
 	${DOCKRUN} bash ./scripts/check.sh
 
-todo: prepare
+todo:
 	${DOCKRUN} bash ./scripts/todo.sh
 
-test: prepare format check
-	${DOCKRUN} bash ./scripts/test.sh
+db_seed:
+	docker run --name mysql-db -e MYSQL_ALLOW_EMPTY_PASSWORD=yes -p 3306:3306 -d mysql:5.6
+	docker run --name mongo-db -p 27015:27015 -d mongo:4.2.0
+	docker cp apna_school.sql mysql-db:apna_school.sql
+	docker cp apna_school.mongo mongo-db:apna_school.mongo
+
+db_start:
+	docker exec mysql-db sh -c 'mysql -uroot < apna_school.sql'
+	docker exec mongo-db sh -c 'mongo < apna_school.mongo'
+
+db_stop:
+	docker stop mysql-db
+	docker stop mongo-db
+	docker rm -f mysql-db
+	docker rm -f mongo-db
+
+test: check db_seed db_start db_stop
```

Comment:
The error you are facing could be because `db_stop` is a pre step to `db_test` Which is incorrect. db stop is post stop. After the tests have been concluded.